gradoop-flink/src/main/java/org/gradoop/flink/algorithms/gelly/GradoopGellyAlgorithm.java

```diff
@@ -78,20 +89,19 @@ public LogicalGraph execute(LogicalGraph graph) {
    * @param graph Gradoop Graph.
    * @return Gelly Graph.
    */
-  public Graph<GradoopId, VV, EV> transformToGelly(LogicalGraph graph) {
-    DataSet<Vertex<GradoopId, VV>> gellyVertices = graph.getVertices().map(toGellyVertex);
-    DataSet<Edge<GradoopId, EV>> gellyEdges = graph.getEdges().map(toGellyEdge);
-    return Graph.fromDataSet(gellyVertices, gellyEdges,
-      graph.getConfig().getExecutionEnvironment());
+  public Graph<GradoopId, VV, EV> transformToGelly(LG graph) {
+    DataSet<org.apache.flink.graph.Vertex<GradoopId, VV>> gellyVertices = graph.getVertices()
```

Comment:
I prefer imports for Gradoop elements and in case of collision qualified names for the external element, to prevent confusion. I forgot the import for the Gradoop elements